packages/@react-spectrum/listbox/stories/ListBox.stories.tsx

```diff
@@ -127,6 +140,18 @@ storiesOf('ListBox', module)
       </ListBox>
     )
   )
+  .add(
```

Comment:
do we need all the stories if we have all the tests?

Reply:
Eh, I guess not but good to have just for any unexpected cross browser cases (not that I would expect there to be any for this issue specifically)